tests/mr/device/mr_ref_test.hpp

```diff
@@ -45,8 +45,7 @@
 #include <random>
 #include <utility>
 
-using resource_ref       = cuda::mr::resource_ref<cuda::mr::device_accessible>;
-using async_resource_ref = rmm::device_async_resource_ref;
+using resource_ref = cuda::mr::resource_ref<cuda::mr::device_accessible>;
```

Comment:
We could add a corresponding `rmm::device_resource_ref` in `resource_ref.hpp`? I didn't add it in the original PR because I didn't have an immediate use for it.

Reply:
This is only ever used once here, so I believe it does not add too much value to add that alias globally